lib/ansible/plugins/filter/password_hash.yml

```diff
@@ -17,7 +17,7 @@ DOCUMENTATION:
       description: Hashing algorithm to use.
       type: string
       default: sha512
-      choices: [ md5, blowfish, sha256, sha512, bcrypt ]
+      choices: [ md5, blowfish, sha256, sha512, bcrypt, argon2 ]
```

Comment:
Any change here would also need to be reflected in `lib/ansible/utils/encrypt.py` to properly configure the defaults for this algorithm.